Corrected path to iris_sklearn.py

DaveVoyles opened the pull request:
Tried running this on an ubuntu VM with *.\iris_sklearn.py* and console returns:

```
az ml experiment submit -c dv-data-sci-ubuntu .\iris_sk_learn.py
File to run does not exist.
```

Removed *./* and it works:

[Link to image of it working in console](https://www.dropbox.com/s/zbbn55jsu32eef7/Screenshot%202018-01-19%2020.12.57.png?dl=0)

azurecla commented:
Hi __@sahaj08__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sahajs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess

DaveVoyles committed:
Corrected path to iris_sklearn.py

Tried running this on an ubuntu VM with *.\iris_sklearn.py* and console returns:

```
az ml experiment submit -c dv-data-sci-ubuntu .\iris_sk_learn.py
File to run does not exist.
```

Removed *./* and it works:

[Link to image of it working in console](https://www.dropbox.com/s/zbbn55jsu32eef7/Screenshot%202018-01-19%2020.12.57.png?dl=0)
